draft-ietf-tls-external-psk-importer.md

```diff
@@ -91,15 +90,14 @@ when, and only when, they appear in all capitals, as shown here.
 Key importers mirror the concept of key exporters in TLS in that they diversify a key
 based on some contextual information before use in a connection. In contrast to key exporters,
 wherein differentiation is done via an explicit label and context string, the key importer
-defined herein uses an optional context string along with a target protocol and KDF
-identifier to differentiate an external PSK into one or more PSKs for use.
+defined herein diversifies external one PSK into one or more PSKs for use via a target protocol,
+KDF identifier, and optional context string. Additionally, the resulting PSK binder key is modified
+with a new derivation label to prevent confusion with non-imported PSKs.
 
 Imported keys do not require negotiation for use, as a client and server will not agree upon
-identities if not imported correctly. Thus, importers induce no protocol changes with
-the exception of expanding the set of PSK identities sent on the wire. Endpoints may
-incrementally deploy PSK importer support by offering non-imported keys for TLS versions
-prior to TLS 1.3. Non-imported and imported PSKs are distinct since their identities are
-different on the wire. See {{rollout}} for more details.
+identities if not imported correctly. Endpoints may incrementally deploy PSK importer support
```

Comment:
```suggestion
identities if not imported correctly. Endpoints MAY incrementally deploy PSK importer support
```

Reply:
I didn't apply this suggestion since this is not about the import behavior per-se, so I didn't want to apply any type of normative hint.